plugin.go

```diff
@@ -44,6 +44,9 @@ func (p Plugin) Exec() error {
 
 	var commands []*exec.Cmd
 	remote := p.Config.Remote
+	if len(p.Config.Secrets) != 0 {
+		exportSecrets(p.Config.Secrets)
```

Comment:
Another reason I went with this approach is that the remote config command for terraform can take in a secret and access key but it will place them in plain text into the remote state. That is very undesirable.